common/lib/dependabot/pull_request_creator.rb

```diff
@@ -126,7 +126,7 @@ def gitlab_creator
         author_details: author_details,
         labeler: labeler,
         approvers: reviewers,
-        assignee: assignees&.first,
+        assignees: assignees,
```

Comment:
👋 I think this change will break the other pull request creators, could you update these to pick the first one?